# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1462313987 25200
#      Tue May 03 15:19:47 2016 -0700
# Node ID 5e083d07ba35f423983e191bbe37ebecca4d8043
# Parent  9631aac4b63518e621b9752233387f7b5dc9c8c2
maint: add DOCSTRINGS to .hgignore

* .hgignore: Add (libinterp|scripts)/DOCSTRINGS to the ignore list.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -35,8 +35,11 @@ syntax: regexp
 ^doc/\w*/stamp-(vti|\d+)$
 ^doc/\w*/version-\w+\.texi$
 ^doc/interpreter/images\.mk$
 
 # e.g. liboctave/operators/smx-op-inc.mk
 ^liboctave/operators/\w+-op-\w+\.mk$
 
 ^libinterp/dldfcn/module\.mk$
+
+# DOCSTRINGS files are built in the source tree
+^(libinterp|scripts)/DOCSTRINGS$
